gcloud/_apitools/http_wrapper.py

```diff
@@ -165,7 +163,7 @@ def ProcessContentRange(content_range):
             start, _, end = byte_range.partition('-')
             return int(end) - int(start) + 1
 
-        if '-content-encoding' in self.info and 'content-range' in self.info:
+        if 'content-encoding' in self.info and 'content-range' in self.info:
```

Comment:
RE: "Type". Oy. I figured it was some suffix.

RE: Py2/Py3 annoyance. Should we comment to this effect?